corehq/apps/campaign/forms.py

```diff
@@ -182,8 +183,10 @@ class Meta(DashboardWidgetBaseForm.Meta):
 
     def __init__(self, domain, *args, **kwargs):
         super().__init__(domain, *args, **kwargs)
-        self.fields['case_type'].choices = self._get_case_types()
+        case_type_choices = [('', '')]
+        case_type_choices.extend(self._get_case_type_choices())
+        self.fields['case_type'].choices = case_type_choices
 
-    def _get_case_types(self):
+    def _get_case_type_choices(self):
         case_types = sorted(get_case_types_for_domain(self.domain))
         return [(case_type, case_type) for case_type in case_types]
```

Comment:
Nit: Now that this is called `_get_case_type_choices()`, I would put all the choices in here, to simplify the calling code. Maybe something like:
```python
    def _get_case_type_choices(self):
        case_types = sorted(get_case_types_for_domain(self.domain))
        choices = [(case_type, case_type) for case_type in case_types]
        return [('', '---')] + choices
```

Reply:
https://github.com/dimagi/commcare-hq/pull/36096/commits/6c08ca15ba7600e2ba065b7f14c034e7f20fea00